packages/inference/src/providers/replicate.ts

```diff
@@ -13,9 +13,55 @@ type ReplicateId = string;
  * 	-H "Authorization: Bearer $REPLICATE_API_TOKEN" \
  * 	'https://api.replicate.com/v1/models'
  * ```
+ * 
+ * See also https://github.com/replicate/all-the-public-replicate-models
  */
 export const REPLICATE_MODEL_IDS: Record<ModelId, ReplicateId> = {
 	/** text-to-image */
+	"black-forest-labs/FLUX.1-Canny-dev": "black-forest-labs/flux-canny-dev",
+	"black-forest-labs/FLUX.1-Depth-dev": "black-forest-labs/flux-depth-dev",
+	"black-forest-labs/FLUX.1-dev": "black-forest-labs/flux-dev",
+	"black-forest-labs/FLUX.1-Fill-dev": "black-forest-labs/flux-fill-dev",
+	"black-forest-labs/FLUX.1-Redux-dev": "black-forest-labs/flux-redux-dev",
 	"black-forest-labs/FLUX.1-schnell": "black-forest-labs/flux-schnell",
+	"ByteDance/Hyper-SD": "bytedance/hyper-flux-16step",
 	"ByteDance/SDXL-Lightning": "bytedance/sdxl-lightning-4step",
-};
+	"dataautogpt3/ProteusV0.3": "datacte/proteus-v0.3",
+	"Efficient-Large-Model/Sana_1600M_1024px": "nvidia/sana",
+	"fal/AuraFlow": "fofr/aura-flow",
+	"KwaiVGI/LivePortrait": "fofr/expression-editor",
+	"playgroundai/playground-v2.5-1024px-aesthetic": "playgroundai/playground-v2.5-1024px-aesthetic",
+	"SG161222/Realistic_Vision_V5.1_noVAE": "lucataco/realistic-vision-v5.1",
+	"SG161222/RealVisXL_V3.0": "fofr/realvisxl-v3-multi-controlnet-lora",
+	"stabilityai/stable-diffusion-3.5-large-turbo": "stability-ai/stable-diffusion-3.5-large-turbo",
+	"stabilityai/stable-diffusion-3.5-large": "stability-ai/stable-diffusion-3.5-large",
+	"stabilityai/stable-diffusion-3.5-medium": "stability-ai/stable-diffusion-3.5-medium",
+	"stabilityai/stable-diffusion-xl-base-1.0": "stability-ai/sdxl",
+	"XLabs-AI/flux-controlnet-collections": "xlabs-ai/flux-dev-controlnet",
+
+	/** image tools */
+	"Carve/tracer_b7": "lucataco/remove-bg",
```

Comment:
this HF model page doesn't have an identified "pipeline type" but we could open a PR to add one (image-to-image?)

BTW our list of tasks is here: https://huggingface.co/tasks

Reply:
image-to-image makes sense to me. 👍🏼